migrate/sqlschema/inspector.go

```diff
@@ -0,0 +1,200 @@
+package sqlschema
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/uptrace/bun"
+	"github.com/uptrace/bun/schema"
+	orderedmap "github.com/wk8/go-ordered-map/v2"
+)
+
+type InspectorDialect interface {
+	schema.Dialect
+	Inspector(db *bun.DB, excludeTables ...string) Inspector
```

Comment:
`excludeTables` should be an option so we can extend this later.

`Inspector(db *bun.DB, options ...sqlschema.InspectorOption) Inspector`

Reply:
This is a duplicate of the suggestion above.